src/kv/cached.rs

```diff
@@ -252,9 +277,9 @@ async fn handle_entry(
     claims: Arc<RwLock<ClaimsMap>>,
 ) {
     if entry.key.starts_with(LINKDEF_PREFIX) {
-        handle_linkdef(entry, linkdefs).await;
+        handle_linkdef(entry, linkdefs).in_current_span().await;
```

Comment:
Any reason for `in_current_span` vs instrumenting these functions?

Reply:
Yep, this was so it all shows up as being part of the main cache kv watcher. I didn't think the distinction between the two functions really mattered in this case